pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -580,6 +581,52 @@ func GetTaskRunName(taskRunsStatus map[string]*v1beta1.PipelineRunTaskRunStatus,
 	return kmeta.ChildName(prName, fmt.Sprintf("-%s", ptName))
 }
 
```

Comment:
thanks @jerop for this, I am trying to understand how this existing function will be changed (if at all) after we introduce these new functions. 🙏

Reply:
@pritidesai the existing function will not be changed

see related PR: https://github.com/tektoncd/pipeline/pull/4951